--- origsrc/dwm-6.0/Makefile	2011-12-19 09:02:46.000000000 -0600
+++ src/dwm-6.0/Makefile	2013-07-29 19:24:22.906104800 -0500
@@ -16,7 +16,7 @@ options:
 
 .c.o:
 	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	@${CC} -c ${CFLAGS} ${CPPFLAGS} $<
 
 ${OBJ}: config.h config.mk
 
@@ -26,7 +26,7 @@ config.h:
 
 dwm: ${OBJ}
 	@echo CC -o $@
-	@${CC} -o $@ ${OBJ} ${LDFLAGS}
+	@${CC} -o $@ ${OBJ} ${LDFLAGS} ${LIBS}
 
 clean:
 	@echo cleaning
--- origsrc/dwm-6.0/config.mk	2011-12-19 09:02:46.000000000 -0600
+++ src/dwm-6.0/config.mk	2013-07-29 19:23:59.391759900 -0500
@@ -21,9 +21,9 @@ LIBS = -L/usr/lib -lc -L${X11LIB} -lX11
 # flags
 CPPFLAGS = -DVERSION=\"${VERSION}\" ${XINERAMAFLAGS}
 #CFLAGS = -g -std=c99 -pedantic -Wall -O0 ${INCS} ${CPPFLAGS}
-CFLAGS = -std=c99 -pedantic -Wall -Os ${INCS} ${CPPFLAGS}
+CFLAGS = -std=c99 -pedantic -Wall -Os ${INCS}
 #LDFLAGS = -g ${LIBS}
-LDFLAGS = -s ${LIBS}
+LDFLAGS = -s
 
 # Solaris
 #CFLAGS = -fast ${INCS} -DVERSION=\"${VERSION}\"
